engine/BUILD.bazel

```diff
@@ -0,0 +1,66 @@
+load("//:version.bzl", "PROJECT_VERSION")
+
+cc_library(
```

Comment:
if you name this library engine you can consume it via //engine.  If not you have to use the following //engine:enginelib

Reply:
I know. I do not want it to be used outside just yet. If anything, I'd want `cloe-engine` binary target to be named just `engine`, so that it can be run more easily. Until we decide, better to not have a default yet.